components/skeleton/skeleton.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import useTheme from "../use-theme";
+
+interface Props {
+    width?: number
+    squared?: boolean
+    rounded?: boolean
+    component?: keyof JSX.IntrinsicElements
+    className?: string
+    show?: boolean
+    minHeight?: number
+    animate?: boolean
+    height?: number
+}
+
+const defaultProps = {
+    squared: false,
+    rounded: false,
+    component: 'span' as keyof JSX.IntrinsicElements,
+    className: '',
+    show: false,
+    minHeight: 24,
+    animate: true,
+}
+
+type NativeAttrs = Omit<React.HTMLAttributes<any>, keyof Props>
+export type SkeletonProps = Props & NativeAttrs
+
+const Skeleton: React.FC<React.PropsWithChildren<SkeletonProps>> = ({
+                                                                        component,
```

Comment:
Not quite sure if you're using `prettier`.

Reply:
I ran the `lint` script without a change, I'll have a look why this happened :D